packages/get-platform/src/test-utils/jestContext.ts

```diff
@@ -52,6 +52,10 @@ export const jestContext = {
     beforeEach(() => {
       const originalCwd = process.cwd()
 
+      c.mocked = c.mocked ?? {
+        cwd: process.cwd(),
+      }
```

Comment:
I imagine this is your editor formatting the file automatically?

Reply:
I moved this up because while debugging tests I ran into some cases where things blew up before this `c.mocked` was even set correctly. So the `afterEach` would not correctly clean up and cause very confusing error messages in further tests. So I moved this up to run first thing in the beforEach.